pimd/pim_nht.c

```diff
@@ -258,12 +252,10 @@ void pim_nht_bsr_del(struct pim_instance *pim, struct in_addr addr)
 	 * is 0.0.0.0 as that the BSR has not been registered
 	 * for tracking yet.
 	 */
-	if (addr.s_addr == INADDR_ANY)
+	if (pim_addr_is_any(addr))
 		return;
 
-	lookup.rpf.rpf_addr.family = AF_INET;
-	lookup.rpf.rpf_addr.prefixlen = IPV4_MAX_BITLEN;
-	lookup.rpf.rpf_addr.u.prefix4 = addr;
+	pim_addr_to_prefix(&lookup.rpf.rpf_addr, addr);
 
 	pnc = hash_lookup(pim->rpf_hash, &lookup);
 
```

Comment:
There are two format strings a few lines below (unchanged) that need to be changed to `%pPA` instead of `%pI4`.

Reply:
Thanks @eqvinox. This seems to newly added, It is fixed now.